src/tokens/saga.test.ts

```diff
@@ -343,9 +341,7 @@ describe(fetchTokenBalancesForAddressByTokenId, () => {
     })
     expect(mockFetch.mock.calls.length).toEqual(1)
     expect(mockFetch.mock.calls[0][0]).toEqual(
-      encodeURI(
-        'https://api.alfajores.valora.xyz/getWalletBalances?address=some-address&networkIds[]=celo-alfajores&networkIds[]=ethereum-sepolia'
-      )
+      'https://api.alfajores.valora.xyz/getWalletBalances?address=some-address&networkIds=celo-alfajores%2Cethereum-sepolia'
```

Comment:
supernit: the URL encoding seems to be a byproduct here. what we really want is to control that the comma is here. that's why I would prefer to describe the test case with the real comma instead of `%2C` gibberish.